frame/staking/src/tests.rs

```diff
@@ -1600,7 +1597,11 @@ fn on_free_balance_zero_stash_removes_validator() {
 fn on_free_balance_zero_stash_removes_nominator() {
 	// Tests that nominator storage items are cleaned up when stash is empty
 	// Tests that storage items are untouched when controller is empty
-	ExtBuilder::default().existential_deposit(10).build_and_execute(|| {
+	ExtBuilder::default()
+	.existential_deposit(10)
+	.min_nominator_bond(10)
+	.min_validator_bond(10)
+	.build_and_execute(|| {
```

Comment:
```suggestion
	    .existential_deposit(10)
	    .min_nominator_bond(10)
	    .min_validator_bond(10)
	    .build_and_execute(|| {
```

Might need to do that in your editor since I can't figure out how to use tabs here

Reply:
The only good way based on my experience is to copy another tab and paste it :D